### DISCUSSION TITLE
How to convert []byte -> JSON?

### DISCUSSION THREAD

**Discussion by jamesli2021** [sqlc-dev/sqlc#2477] [Q&A]
I'm struggle with sqlc using pgx v5, now that JSON return when I have a query with both output:

col1 []byte
col2 []string

Despite spent 12 hours and still get garbarge text, how do I properly convert all slice byte and string to JSON and return to frontend e.g. React or HTML that is not manage by Go app? pgx v4 use pgtype.JSON was a better DX.

**Comment by andrewmbenton** [comment]
Sorry, I don't quite understand the question. Can you share some example code that sqlc is generating with `pgx/v5` and where the conversion to JSON is causing a problem?
